cluster/examples/kubernetes/ceph/csi/rbac/rbd/csi-provisioner-rbac.yaml

```diff
@@ -37,12 +37,6 @@ rules:
   - apiGroups: [""]
     resources: ["nodes"]
     verbs: ["get", "list", "watch"]
-  - apiGroups: ["storage.k8s.io"]
```

Comment:
the volumeattachments are needed for the rbd driver anymore?

Reply:
@travisn we have a multiple   `volumeattachment` at https://github.com/rook/rook/blob/master/cluster/examples/kubernetes/ceph/csi/rbac/rbd/csi-provisioner-rbac.yaml#L35 and https://github.com/rook/rook/blob/master/cluster/examples/kubernetes/ceph/csi/rbac/rbd/csi-provisioner-rbac.yaml#L41  so am  removing the duplicate